redbeat/schedulers.py

```diff
@@ -189,23 +189,27 @@ def __init__(self, app, **kwargs):
         super(RedBeatScheduler, self).__init__(app, **kwargs)
 
     def setup_schedule(self):
-        # cleanup old static entries
+        # cleanup old static schedule entries
         client = redis(self.app)
-        previous = client.smembers(self.app.conf.REDBEAT_STATICS_KEY)
-        current = set(self.app.conf.CELERYBEAT_SCHEDULE.keys())
-        removed = previous - current
+        previous = set(key for key in client.smembers(self.app.conf.REDBEAT_STATICS_KEY))
```

Comment:
No. ;) There used to be a `key.decode()` here, which isn't needed anymore. Could be written like this now:

    previous = set(client.smembers(self.app.conf.REDBEAT_STATICS_KEY))